thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/alertFilterAutotune/AlertFilterAutotuneFactory.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.thirdeye.anomalydetection.alertFilterAutotune;
+
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFeedbackDTO;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.detector.function.AnomalyFunctionFactory;
+import com.linkedin.thirdeye.detector.function.BaseAnomalyFunction;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Properties;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Created by ychung on 2/8/17.
+ */
+public class AlertFilterAutotuneFactory {
+  private static Logger LOGGER = LoggerFactory.getLogger(AlertFilterAutotuneFactory.class);
+  private final Properties props;
+
+  public AlertFilterAutotuneFactory(String functionConfigPath) {
+    props = new Properties();
+    try {
+      InputStream input = new FileInputStream(functionConfigPath);
+      loadPropertiesFromInputStream(input);
+    } catch (FileNotFoundException e) {
+      LOGGER.error("File {} not found", functionConfigPath, e);
```

Comment:
Add more information for debugging purpose. For example, what is the usage of this file.